backend/data/src/main/kotlin/io/tolgee/model/slackIntegration/SlackMessageInfo.kt

```diff
@@ -0,0 +1,18 @@
+package io.tolgee.model.slackIntegration
+
+import io.tolgee.model.StandardAuditModel
+import jakarta.persistence.*
+import org.hibernate.annotations.ColumnDefault
+
+@Entity
+class SlackMessageInfo(
+  @ManyToOne(fetch = FetchType.LAZY)
+  var slackMessage: SavedSlackMessage = SavedSlackMessage(),
+  var langTag: String = "",
+) : StandardAuditModel() {
+  @Enumerated(EnumType.STRING)
+  @ColumnDefault("'GLOBAL'")
```

Comment:
```suggestion
  @ColumnDefault("GLOBAL")
```

I think it should work without the apos.